lib/calendar.ts

```diff
@@ -1958,6 +2197,13 @@ const helperJapanese: NonIsoHelperBase = ObjectAssign(
   } as Partial<NonIsoHelperBase>
 );
 
+interface ChineseMonthInfo {
+  [key: string]: { monthIndex: number; daysInMonth: number };
+}
+interface ChineseDraftMonthInfo {
+  [key: string]: { monthIndex: number; daysInMonth?: number };
+}
```

Comment:
Also not sure if this is valid TypeScript but can we do something like `interface ChineseMonthInfo = Required<ChineseDraftMonthInfo>`?